src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/RestoreTelemetryEvent.cs

```diff
@@ -24,7 +24,7 @@ public class RestoreTelemetryEvent : ActionEventBase
             double duration) : base(RestoreActionEventName, operationId, projectIds, startTime, status, packageCount, endTime, duration)
         {
             base[nameof(OperationSource)] = source;
-            base[(nameof(NoOpProjectsCount))] = noOpProjectsCount;
+            base[nameof(NoOpProjectsCount)] = noOpProjectsCount;
```

Comment:
`NoOpProjectsCount` wasn't being passed until now or it didn't make any diff?

Reply:
I just removed the extra brackets around it. just a clean up diff.